docs/getting-started-kubernetes.rst

```diff
@@ -857,19 +857,87 @@ The ``pod.yaml`` configuration file for such a deployment:
 SR-IOV Network Operator Deployment – Parallel Node Configuration for SR-IOV
 ---------------------------------------------------------------------------
 
-.. warning:: This is a Tech Preview feature, which is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
+.. warning:: This is a feature is supported only for Vanilla Kubernetes deployments with SR-IOV Network Operator.
 
-To apply SriovNetworkNodePolicy on several nodes in parallel, specify the ``maxParallelConfiguration`` option in the SriovOperatorConfig CRD:
+To apply configure SR-IOV on several nodes in parallel, the ``SriovNetworkPoolConfig`` CRD and specify number or percent of unavailable nodes at the moment:
```

Comment:
suggestion:
To apply SR-IOV configuration on several nodes in parallel, create a ``SriovNetworkPoolConfig`` CR and specify the maximum number or percentage of nodes that can be unavailable at the same time:

Reply:
done